temporalcli/commandsgen/commands.yml

```diff
@@ -3773,14 +3763,12 @@ commands:
           Assume update inputs are base64-encoded and attempt to decode them.
 
   - name: temporal workflow execute-update-with-start
-    summary: Send an Update and wait for it to complete (Experimental)
+    summary: Send an Update-With-Start and wait for it to complete
     description: |
       Send a message to a Workflow Execution to invoke an Update handler, and wait for
       the update to complete. If the Workflow Execution is not running, then a new workflow
       execution is started and the update is sent.
 
-      Experimental.
```

Comment:
I think this is technically experimental same as `start-update-with-start` but I don't mind if we remove it